qiskit/pulse/instructions/play.py

```diff
@@ -47,6 +47,9 @@ def __init__(self, pulse: Pulse,
         """
         if not isinstance(pulse, Pulse):
             raise PulseError("The `pulse` argument to `Play` must be of type `library.Pulse`.")
+        if not isinstance(channel, PulseChannel):
```

Comment:
Where did this come up?

Reply:
It's not directly related, but I saw it and so wanted to make sure it was addressed.